Build error on 9.3.0

I'm getting the following error when trying to build 9.3.0:

```
CMake Error at CMakeLists.txt:96 (add_library):
  add_library cannot create ALIAS target "llhttp::llhttp" because another
  target with the same name already exists.
```

I haven't investigated yet, just opening the issue for now.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -92,9 +92,7 @@ if(BUILD_STATIC_LIBS)
   add_library(llhttp_static STATIC
       ${llhttp_src}
   )
-  if(BUILD_SHARED_LIBS)
-    add_library(llhttp::llhttp ALIAS llhttp_shared)
-  else()
+  if(NOT BUILD_SHARED_LIBS)
     add_library(llhttp::llhttp ALIAS llhttp_static)
   endif()
   config_library(llhttp_static)

